src/flows/recommendation_api/backfill_hashed_user_preferences.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict, Sequence, List
+from prefect import Flow, Parameter, unmapped, task
+from prefect.executors import LocalDaskExecutor
+from utils.flow import get_flow_name, get_interval_schedule
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from api_clients.athena import AthenaQuery
+from sagemaker.feature_store.inputs import FeatureValue
+from utils import config
+from common_tasks.corpus_candidate_set import load_feature_record
+
+FLOW_NAME = get_flow_name(__file__)
+
+ATHENA_SQL = """
+SELECT 
+    user_id, 
+    updated_at, 
+    preferred_topics
+FROM "sagemaker_featurestore"."development-user-recommendation-preferences-v1-1654826050" 
+limit 10
+"""
+
+HASHED_USER_ID_MAP_QUERY = """
+SELECT 
+    user_id as "USER_ID", 
+    hashed_user_id as "HASHED_USER_ID"
+FROM "ANALYTICS"."DBT"."USERS"
+WHERE user_id = %(user_id)s
+"""
+
+V2_FEATURE_GROUP_NAME = 'development-user-recommendation-preferences-v2'
+
+@task()
+def get_user_topics_pref_rows(df):
+    return df.to_dict('records')
+
+@task()
+def cleanup_v2_user_topics_prefs(v2_record: List[Dict]) -> Sequence[FeatureValue]:
+    return list(filter(lambda x: x != None, v2_record))
+
+@task()
+def build_v2_user_topics_prefs(
+        v1_record: Dict
+) -> Sequence[FeatureValue]:
+
+    query = PocketSnowflakeQuery(
+        database=config.SNOWFLAKE_ANALYTICS_DATABASE,
+        schema=config.SNOWFLAKE_ANALYTICS_DBT_SCHEMA,
+        output_type = OutputType.DICT
+    )
+    hashed_user_id_map = query.run(
+        query=HASHED_USER_ID_MAP_QUERY,
+        data=v1_record,
+    )
+
+    return [
+        FeatureValue('hashed_user_id', hashed_user_id_map[0]['HASHED_USER_ID']),
+        FeatureValue('updated_at', v1_record['updated_at']),
+        FeatureValue('preferred_topics', v1_record['preferred_topics']),
+    ] if len(hashed_user_id_map) else None
```

Comment:
The [fail-fast](https://getpocket.atlassian.net/wiki/spaces/PE/pages/2754740314/Software+Design+Patterns#Fail-fast) approach would be:
```suggestion
    ]
```
The goal is:
- Spend less effort trying to be fault-tolerant
- By letting errors bubble up, errors and their root cause are easier to spot.